spinalcordtoolbox/deepseg/models.py

```diff
@@ -183,6 +183,15 @@
          "thr": None,  # Images are already binarized
          "default": False,
      },
+    "model_seg_canal_t2w": {
```

Comment:
We probably want the task name to be different than the model name. For example, `seg_canal_t2w`.

(Alternatively, if we want to adopt the v7.0 naming scheme now, we can simply go with `canal_t2w` or even `sc_canal_t2w`.)

Reply:
> if we want to adopt the v7.0 naming scheme now

Good idea!

> we can simply go with `canal_t2w` or even `sc_canal_t2w`

From these two, I would vote for `canal_t2w`.

On the other, if we want to make explicit that the canal segmentation incldues not only CSF but also SC, then
`sc_canal_t2w` could probably work as well.

Reply:
Yes I think also for now that canal_t2w is the most explicit name !

Reply:
Done ! task rename "canal_t2w"